### PR TITLE
Better UI + Code

### PR DESCRIPTION
**Pull request by SoyOscarRH**
Here is my proposal for the UI, 
What do you think?

## Example Ipad
drawing

## Example Pixel 2
drawing

### CODE REVIEW

**Comment by SoyOscarRH** [comment]
Examples after the dropdown

Example2

drawing

**Comment by fancast** [comment]
> Examples after the dropdown
> 
> Example2
> 
> drawing

I like this version of the UI. It is very easy to use and interact with.

File: fudge-ratio/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,55 +1,61 @@
...
-let timer;
+let seconds = 0,
+  minutes = 0,
+  hours = 0;
+let timer = null;
 
-// Checks when the start and stop buttons are clicked once the HTML page is loaded.
-window.addEventListener("load", activateListeners);
+// Concatenates a zero to the beginning of a number less than 10.
+const padd = num => (num < 10 ? "0" : "") + num;
```
Comment (fancast) [inline fudge-ratio/src/main/webapp/script.js:12]: I did not know you can create if statements in that way. Very creative!

File: fudge-ratio/src/main/webapp/script.js [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
-}
\ No newline at end of file
+button.addEventListener("click", () => {
+  if (button.textContent === "Start") {
+    const minutesSince = startTimer();
+  } else {
+    stopTimer();
+  }
+
+  button.classList.toggle("stop");
```
Comment (fancast) [inline fudge-ratio/src/main/webapp/script.js:52]: This is a good use of toggle

File: fudge-ratio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,14 +1,95 @@
...
+.dropdown-contents {
+  width: 35vw;
+  font-size: calc(0.8 * var(--base));
+  background-color: rgba(0, 0, 0, 0.1);
+  padding: 0.1rem;
+}
+
+.dropdown-contents:hover {
+  box-shadow: 1px 1px 11px grey;
+}
```
Comment (fancast) [inline fudge-ratio/src/main/webapp/style.css:95]: This looks good to me

**Review by fancast — APPROVED** [review]
This pull request is good to go!

**Pull request merged**